pkg/master/controller/crdregistration/crdregistration_controller.go

```diff
@@ -195,26 +216,25 @@ func (c *crdRegistrationController) handleVersionUpdate(groupVersion schema.Grou
 		return err
 	}
 	for _, crd := range crds {
-		if crd.Spec.Version == groupVersion.Version && crd.Spec.Group == groupVersion.Group {
-			found = true
-			break
+		if crd.Spec.Group != groupVersion.Group {
+			continue
+		}
+		for index, version := range crd.Spec.Versions {
+			if version.Name == groupVersion.Version && version.Served {
```

Comment:
nit: `continue` if it's not true. Makes it easier to read and is more Go idiomatic

Reply:
done